sdk/runanywhere-swift/Sources/RunAnywhere/Features/STT/Services/LiveTranscriptionSession.swift

```diff
@@ -0,0 +1,264 @@
+//
+//  LiveTranscriptionSession.swift
+//  RunAnywhere SDK
+//
+//  High-level API for live/streaming transcription.
+//  Combines audio capture and streaming transcription into a single abstraction.
+//
+
+import Foundation
+import Combine
+
+// MARK: - Live Transcription Session
+
+/// A session for live/streaming speech-to-text transcription.
+///
+/// This provides a high-level API that combines audio capture and streaming
+/// transcription, handling all the complexity internally.
+///
+/// ## Usage
+///
+/// ```swift
+/// // Start live transcription
+/// let session = try await RunAnywhere.startLiveTranscription()
+///
+/// // Listen for transcription updates
+/// for await text in session.transcriptions {
+///     print("Partial: \(text)")
+/// }
+///
+/// // Or use callback style
+/// let session = try await RunAnywhere.startLiveTranscription { text in
+///     print("Partial: \(text)")
+/// }
+///
+/// // Stop when done
+/// await session.stop()
+/// ```
+@MainActor
+public final class LiveTranscriptionSession: ObservableObject {
```

Comment:
Do we have an equivalent batch and live both? This should sit in the /public/ folder just because it is a direct public API